src/cpp/fastdds/xtypes/type_representation/TypeObjectRegistry.hpp

```diff
@@ -189,18 +189,6 @@ class TypeObjectRegistry : public ITypeObjectRegistry
      * @return ReturnCode_t RETCODE_OK if the operation is successful.
      *                      RETCODE_NO_DATA if any given TypeIdentifier is unknown to the registry.
      *                      RETCODE_BAD_PARAMETER if any given TypeIdentifier is not a direct hash.
-     */
-    ReturnCode_t get_direct_hash_type_dependencies(
-            const TypeIdentifierSeq& type_identifiers,
-            std::unordered_set<TypeIdentfierWithSize>& type_dependencies);
-
-    /**
-     * @brief Get the type dependencies of the given type identifiers.
-     *
-     * @param[in] type_identifiers Sequence with the queried TypeIdentifiers.
-     * @param[in out] type_dependencies Unordered set of TypeIdentifiers with related TypeObject serialized size.
-     * @return ReturnCode_t RETCODE_OK if the operation is successful.
-     *                      RETCODE_NO_DATA if any given TypeIdentifier is unknown to the registry.
      *                      RETCODE_BAD_PARAMETER if any given TypeIdentifier is fully descriptive.
```

Comment:
This is redundant with the previous statement. Please, remove.

Reply:
removed in [9449b0e](https://github.com/eProsima/Fast-DDS/pull/4496/commits/9449b0eee57a38c4f9b38a51c31f8bf24d81935f)